sources/api/migration/migrator/tests/data/repository/metadata/1.root.json

```diff
@@ -0,0 +1,71 @@
+{
+  "signed": {
+    "_type": "root",
+    "spec_version": "1.0.0",
+    "consistent_snapshot": true,
+    "roles": {
+      "root": {
+        "keyids": [
+          "c8cd4d9a1436b604690001db35f42c4c07246439ad79649881268e323730eb7c"
+        ],
+        "threshold": 1
+      },
+      "snapshot": {
+        "keyids": [
+          "c63047e43001b3ca29cd6c1ab76b72871fc2c14459cefca5360efa5c3243f276"
+        ],
+        "threshold": 1
+      },
+      "targets": {
+        "keyids": [
+          "5444ff6d392fc17e2b5bea23d2694570f98db5e15591e80144cecf20198ba0a2"
+        ],
+        "threshold": 1
+      },
+      "timestamp": {
+        "keyids": [
+          "29ac0c464b44a29482e1e3c21388844ba2e5f1eb4201e3f7abdd53ba3415032d"
+        ],
+        "threshold": 1
+      }
+    },
+    "version": 1,
+    "keys": {
```

Comment:
Out of curiosity, what was the change to the keys in the force push?

Reply:
> Out of curiosity, what was the change to the keys in the force push?

The keys are throw-away, I'm not saving them somewhere, I have a script that will regenerate the TUF repo whenever I need to make a change, and that includes generating new keys. This problem will go away once the TUF repo is created during the test.

Reply:
Ah, cool.  And we can do that now with https://github.com/awslabs/tough/pull/125/ merged?

Reply:
Addressed by:

> webern force-pushed the webern:pr/signed-migrations branch from f481a2d to 4443441